commands/start.js

```diff
@@ -0,0 +1,127 @@
+const path = require('path')
+const inquirer = require('inquirer')
+const copyRecursively = require('ncp').ncp
+const isDirectoryEmpty = require('../helpers/is-directory-empty')
```

Comment:
to começando a dar uma olhada, to achando show a parte de separar as functions principais. Essa de helpers não poderia estar em um arquivo só ? pq nesse caso eu precisei dar require em vários arquivos diferentes de um `helper`. Ele não poderia ser um `helper.js` e conter essas functions ?

Reply:
boa! acho válido, vou fazer isso ai.